config/activityLog.js

```diff
@@ -124,6 +124,13 @@ module.exports = {
     lastHeartbeat: 'None',
     delta: 'None',
   },
+  missedActivity: {
```

Comment:
is there maybe a more precise word we can use here to refer to what was missed? "activity" seems vague